dessia_common/core.py

```diff
@@ -557,13 +557,20 @@ def zip_settings(self):
         Excel file stream generated by calling 'to_xlsx_stream' method.
         JSON file stream generated by calling 'save_to_stream' method.
         """
-        excel_stream = dcf.BinaryFile("excel_export.xlsx")
-        self.to_xlsx_stream(excel_stream)
-
-        json_stream = dcf.JsonFile("json_export.json")
-        self.save_to_stream(json_stream)
+        list_stream = []
```

Comment:
Avoid stating types in variable names, consider `streams`

Reply:
done !